forum/qa-content/qa-page.js

```diff
@@ -785,6 +785,7 @@ function qa_ajax_error()
             const setCursorToAnnotationEnd = ( editor, ckeTxt ) => {
                 editor.focus();
 
+                // TODO: handle 'getRanges of null' error

```

Comment:
komentarz todo, albo do zrobienia, albo usunąć ten komentarz jeśli już jest zrobione

Reply:
To jest komentarz do błędu, który odnosi się do ficzera adnotacji (nie związany z tym PR), prawdopodobnie występuje też na produkcji. Jeśli dobrze pamiętam, to adnotacje czasem nie działają poprawnie, więc najprawdopodobniej błąd, do którego odnosi się komentarz będzie naprawiony w osobnym PR dotyczącym refaktoryzacji właśnie tamtego ficzera.

Reply:
ok